pkg/ocm/helpers.go

```diff
@@ -518,7 +518,7 @@ func (c *Client) GetPolicies(policyType string) (map[string]string, error) {
 		return m, handleErr(accountRolePoliciesResponse.Error(), err)
 	}
 	accountRolePoliciesResponse.Items().Each(func(awsPolicy *cmv1.AWSSTSPolicy) bool {
-		m[awsPolicy.ID()] = awsPolicy.Details()
+		m[awsPolicy.ID()] = awsPolicy
```

Comment:
This is a preparation for managed policies, where we need the `ARN` attribute as well, 
therefore returning the whole struct instead of only the details field.